libraries/AP_Logger/LogStructure.h

```diff
@@ -625,6 +625,8 @@ struct PACKED log_VisualPosition {
     float roll;     // degrees
     float pitch;    // degrees
     float yaw;      // degrees
+    float pos_err;
+    float ang_err;
```

Comment:
Is the unit radians? Please add it here.

Reply:
I forget it. Thank you for reminding me